src/Controls/src/Core/Picker.cs

```diff
@@ -11,14 +11,8 @@
 namespace Microsoft.Maui.Controls
 {
 	/// <include file="../../docs/Microsoft.Maui.Controls/Picker.xml" path="Type[@FullName='Microsoft.Maui.Controls.Picker']/Docs" />
-	public partial class Picker : View, IFontElement, ITextElement, ITextAlignmentElement, IElementConfiguration<Picker>
+	public partial class Picker : InputView, IFontElement, ITextAlignmentElement, IElementConfiguration<Picker>
```

Comment:
I think Picker is a input control, please let me know if that is wrong and i need to add the property to the interface instead